Userland/Games/Minesweeper/Field.cpp

```diff
@@ -112,12 +112,12 @@ Field::Field(GUI::Label& flag_label, GUI::Label& time_label, GUI::Button& face_b
     , m_time_label(time_label)
     , m_on_size_changed(move(on_size_changed))
 {
-    m_timer = add<Core::Timer>();
-    m_timer->on_timeout = [this] {
-        ++m_time_elapsed;
-        m_time_label.set_text(String::formatted("{}.{}", m_time_elapsed / 10, m_time_elapsed % 10));
-    };
-    m_timer->set_interval(100);
+    m_timer = Core::Timer::create_repeating(
+        100, [this] {
```

Comment:
If it only shows whole seconds, this can update every 1000 ms instead can't it?

Reply:
oh, sure.. i kind of forgot about it :sweat_smile: